src/components/SignInForm.jsx

```diff
@@ -16,7 +16,14 @@ const SignIn = (props, routeProps) => {
 		setFieldType(fieldType === 'Password' ? 'text' : 'Password');
 	};
 
-
+	if (formIncomplete(props) === true || props.values.ContactID === null) {
```

Comment:
not know about about this function, it would probably better be served to be named `isFormValid` but not blocking for that.

```js
if (formIncomplete(props) || !props.values.ContactID) {
```

the second part will handle an empty string, and undefined as well

Reply:
added check for undefined as requested